scripts/components/map.component.js

```diff
@@ -64,4 +70,9 @@ export default class {
     });
     return topoLayer;
   }
+
+  _onToggleBasemap() {
+    document.querySelector('.c-basemap-options').classList.toggle('-open');
+    document.querySelector('.c-map').classList.toggle('-moved');
```

Comment:
It would be ideal here if instead you would dispatch an action, and make the "layer selector" visible/not-visible part of the store.